genai-perf/tests/test_llm_profile_data_parser.py

```diff
@@ -136,6 +145,8 @@ def test_triton_llm_profile_data(self, mock_read_write: pytest.MonkeyPatch) -> N
         assert metrics.output_token_throughputs == pytest.approx(ott)
         assert metrics.output_sequence_lengths == [3, 6]
         assert metrics.input_sequence_lengths == [3, 4]
+        gp = [1 / ns_to_sec(10)]
```

Comment:
I recommend writing out the whole variable here. It's not immediately clear that gp == goodput.

Reply:
Updated according to suggestions from above reviews, except for discussed ones.